luametaplain.ini

```diff
@@ -7,6 +7,7 @@
 % \input luatexconfig.tex
 % Activate primitives
 \input luatexiniconfig.tex
+\input luametatexconfig.tex
```

Comment:
Can you replace the commented `luatexconfig.tex`? Then the loading order is compatible with plain LuaTeX.

Reply:
Fixed.